src/mat2.js

```diff
@@ -291,7 +291,7 @@ export function str(a) {
  * @returns {Number} Frobenius norm
  */
 export function frob(a) {
-  return(Math.sqrt(Math.pow(a[0], 2) + Math.pow(a[1], 2) + Math.pow(a[2], 2) + Math.pow(a[3], 2)))
+  return(Math.hypot(...a))
```

Comment:
I would prefer to not use the spread operator, as I'm not entirely sure about it's performance.

Reply:
that ok, how about Math.hypot to replace Math.sqrt(Math.pow)

Reply:
Yes, of course. That's pretty cool!